components/CreateCard.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
```

Comment:
Still need to import react ? Genuine question have no idea :D !

Reply:
Yes sadly you have to import React in every file!